doc/lua_api.txt

```diff
@@ -2051,7 +2068,8 @@ Elements
 * Textual password style field; will be sent to server when a button is clicked
 * When enter is pressed in field, fields.key_enter_field will be sent with the
   name of this field.
-* Fields are a set height, but will be vertically centred on `H`
+* Fields are a set height, but will be vertically centred on `H` with the old
+  system.  With the new coordinate system, `H` will modify the height.
```

Comment:
Suggestion:
With the old coordinate system, fields are…. With the new system, ….
This might be clearer.